bin/check_samplesheet.py

```diff
@@ -79,9 +79,9 @@ def check_samplesheet(file_in, file_out):
                 if reads:
                     if reads.find(" ") != -1:
                         print_error("Fastq file contains spaces!", "Line", line)
-                    if not reads.endswith(".fastq.gz") and not reads.endswith(".fq.gz"):
+                    if not reads.endswith(".fastq.gz") and not reads.endswith(".fq.gz") and not reads.endswith(".fastq") and not reads.endswith(".fq"):
```

Comment:
hmm, does the underlying modules work with both compressed and regular files?

Reply:
yes

Reply:
I don't have an explicit test for this however, but I have tested it separately with some of Peter's data that he didn't compress